partitioner/src/main/java/io/confluent/connect/storage/partitioner/DailyPartitioner.java

```diff
@@ -30,20 +30,26 @@
 
   @Override
   public void configure(Map<String, Object> config) {
-    long partitionDurationMs = TimeUnit.HOURS.toMillis(24);
+    final long partitionDurationMs = TimeUnit.HOURS.toMillis(24);
```

Comment:
this is here because the variable is declared too far away from where it is actually used.

i guess the other option would be to move this variable down lower